doc/march_packages/march_input_device.rst

```diff
@@ -106,7 +104,8 @@ Start the software
     roslaunch march_launch serial_connection.launch
 
 - Make sure the batteries are charged and in the input device.
-- Press on button, which is located on the electronics holder.
+- Make sure the jumper cap in the electronics holder is placed so the input device is powered by the batteries rather than via a USB cable.
```

Comment:
Is a jumper cap a known term? I'm not sure what it means

Reply:
Depends on the reader. IMO this page is not the place to explain what it is (same as not explaining what a 'port' is, which is earlier on this page). Google tells you the answer quickly enough. I could make the word link to something (e.g. google images result, a confluence page on the ipd). What do you suggest?